internal/application/ingresses.go

```diff
@@ -84,7 +94,10 @@ func SyncIngresses(ctx context.Context, cluster *kubernetes.Cluster, appRef mode
 		}
 		log.Info("creating app ingress", "org", appRef.Org, "app", appRef.Name, "", desiredRoute)
 
-		ing := newAppIngress(appRef, desiredRoute, username)
+		route := routes.FromString(desiredRoute)
+		ingressName := names.IngressName(fmt.Sprintf("%s-%s", appRef.Name, route))
+		ing := route.ToIngress(ingressName)
```

Comment:
```suggestion
		ingress := route.ToIngress(ingressName)
```

I.e. nicer name ? Would need updates where used, of course.

Reply:
07024aa1f3842da96c806101142960a3b20e4c24